x/dsmr/certificate.go

```diff
@@ -66,8 +58,33 @@ func (c *ChunkCertificate) Bytes() []byte {
 	return bytes
 }
 
-func (c *ChunkCertificate) Verify(_ context.Context, _ interface{}) error {
-	return c.Signature.Verify()
+func (c *ChunkCertificate) Verify(
+	ctx context.Context,
+	networkID uint32,
+	chainID ids.ID,
+	pChainState validators.State,
+	pChainHeight uint64,
+	quorumNum uint64,
+	quorumDen uint64,
```

Comment:
I think Aaron mentioned we wanted to support arbitrary verification (i.e not just warp). Probably we'll consider supporting this later...